apps/foundation-demo/src/app/core/loading-icon/loading-icon/loading-icon.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: '[loading-icon]',
+  templateUrl: './loading-icon.component.html',
+  styleUrls: ['./loading-icon.component.scss'],
+  host: {'class': 'loading-icon'},
+  encapsulation: ViewEncapsulation.None
+})
```

Comment:
Can we change the `changeDetection` strategy to `OnPush` for better perf?

Reply:
done